resources/assets/lib/beatmap-discussions/editor-discussion-component.tsx

```diff
@@ -194,17 +195,17 @@ export default class EditorDiscussionComponent extends React.Component<Props> {
   };
 
   nearbyIndicator = (drafts: SlateElement[]) => {
-    if (this.timestamp() == null || this.discussionType() === 'praise') {
+    if (!this.editable() ||  this.timestamp() == null || this.discussionType() === 'praise') {
```

Comment:
this change removes this message

![](https://s.myconan.tk/2022-09/firefox_2022-09-08_15-52-12.png)

Reply:
it's there? ?_?
![image](https://user-images.githubusercontent.com/1694544/189664944-8942a441-4c7b-4cd6-b9ea-ff6606d51cb1.png)

Reply:
that's weird 🤔 maybe I was imagining things. Also I just saw there's an extra space there after `||`.